common/src/main/java/com/wynntils/mc/event/PlayerGhostArmorRenderEvent.java

```diff
@@ -0,0 +1,28 @@
+package com.wynntils.mc.event;
+
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraftforge.eventbus.api.Event;
+
+public class PlayerGhostArmorRenderEvent extends Event {
+
+    private LivingEntity entity;
+    private boolean renderGhostArmor;
+
+    public PlayerGhostArmorRenderEvent(LivingEntity entity) {
+        this.entity = entity;
+        this.renderGhostArmor = true;
+    }
+
+    public LivingEntity getEntity() {
+        return entity;
+    }
+
+    public boolean shouldRenderGhostArmor() {
```

Comment:
I still would rather have the event cancelable, like I mentioned in the last PR

Reply:
Are you intentionally skipping this? Do you need help with it? Or maybe I didn't explain it cleary

Reply:
@YouSoPugly ^